tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -63,14 +63,6 @@ struct RNTupleModelChangeset {
    bool IsEmpty() const { return fAddedFields.empty() && fAddedProjectedFields.empty(); }
 };
 
-/// Merge two RNTuple models. The resulting model will take the description from the left-hand model.
-/// When `rightFieldPrefix` is specified, the right-hand model will be stored in an untyped sub-collection, identified
-/// by the prefix. This way, a field from the right-hand model is represented as `<prefix>.<fieldname>`.
-/// When no prefix is specified, the fields from the right-hand model get added directly to the resulting model.
-///
-/// Note that both models must be frozen before merging.
-std::unique_ptr<RNTupleModel>
-MergeModels(const RNTupleModel &left, const RNTupleModel &right, std::string_view rightFieldPrefix = "");
```

Comment:
This function was added by @enirolf in commit e9573cb4b499977aae33e925136b002bc0a7ed88 "in anticipation for horizontal joining of RNTuples"...

Reply:
Yep, I discussed that with @enirolf. For the moment, that's not needed.